pkg/kudoctl/cmd/convert/convert.go

```diff
@@ -0,0 +1,196 @@
+package convert
+
+import (
+	"fmt"
+	"io"
+	"io/ioutil"
+	"os"
+	"path/filepath"
+	"strings"
+	"syscall"
+
+	"github.com/spf13/cobra"
+	"sigs.k8s.io/yaml"
+
+	kudo "github.com/kudobuilder/kudo/pkg/apis/kudo/v1alpha1"
+	"github.com/kudobuilder/kudo/pkg/util/helm"
+)
+
+// Options are configuration options for this command
+type Options struct {
+	ChartImportPath string
+	OutputPath      string
+}
+
+// DefaultOptions are options with the defaults applied
+var DefaultOptions = &Options{}
+
+// Run runs the convert command
+func Run(cmd *cobra.Command, args []string, options *Options) error {
+	chart, e := helm.LoadChart(options.ChartImportPath)
+	if e != nil {
+		return e
+	}
+	bundle, e := helm.ToBundle(chart)
+	if e != nil {
+		return e
+	}
+	if b := exists(options.OutputPath); b {
```

Comment:
likely we should have options for this...  there are gen times when we way to overwrite and other when we don't.